src/Dev-12/src/test/java/ChromeTestLoginMailRu.java

```diff
@@ -0,0 +1,81 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Class for testing login on mail.ru
+ */
+public class ChromeTestLoginMailRu {
+    private static final String WEBDRIVER_CHROME_DRIVER = "webdriver.chrome.driver";
+    private static final String CHROMEDRIVER_EXE_PATH = ".\\chromedriver\\chromedriver.exe";
+    private static final String WRITE_LETTER = "Написать письмо";
```

Comment:
DRY, to many copy paste and re-usage of the same properties in other classes for other browsers. Create base class for tests and implement these filelds there with public getters.